Deferred is called twice in ooni web ui server

When starting `ooniprobe-agent` I noticed this exception being raised:

```
Unhandled error in Deferred:
Unhandled Error
Traceback (most recent call last):
  File "/usr/local/Cellar/ooniprobe/2.0.1/libexec/vendor/lib/python2.7/site-packages/twisted/internet/defer.py", line 150, in maybeDeferred
    result = f(*args, **kw)
  File "/usr/local/Cellar/ooniprobe/2.0.1/libexec/lib/python2.7/site-packages/ooni/ui/web/server.py", line 138, in notify
    self.lock.acquire().addCallback(self._notify, event)
  File "/usr/local/Cellar/ooniprobe/2.0.1/libexec/vendor/lib/python2.7/site-packages/twisted/internet/defer.py", line 317, in addCallback
    callbackKeywords=kw)
  File "/usr/local/Cellar/ooniprobe/2.0.1/libexec/vendor/lib/python2.7/site-packages/twisted/internet/defer.py", line 306, in addCallbacks
    self._runCallbacks()
--- <exception caught here> ---
  File "/usr/local/Cellar/ooniprobe/2.0.1/libexec/vendor/lib/python2.7/site-packages/twisted/internet/defer.py", line 588, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/usr/local/Cellar/ooniprobe/2.0.1/libexec/lib/python2.7/site-packages/ooni/ui/web/server.py", line 131, in _notify
    assert not d.called, "Deferred is already called"
exceptions.AssertionError: Deferred is already called

Unhandled error in Deferred:
Unhandled Error
Traceback (most recent call last):
  File "/usr/local/Cellar/ooniprobe/2.0.1/libexec/vendor/lib/python2.7/site-packages/twisted/internet/defer.py", line 150, in maybeDeferred
    result = f(*args, **kw)
  File "/usr/local/Cellar/ooniprobe/2.0.1/libexec/lib/python2.7/site-packages/ooni/ui/web/server.py", line 138, in notify
    self.lock.acquire().addCallback(self._notify, event)
  File "/usr/local/Cellar/ooniprobe/2.0.1/libexec/vendor/lib/python2.7/site-packages/twisted/internet/defer.py", line 317, in addCallback
    callbackKeywords=kw)
  File "/usr/local/Cellar/ooniprobe/2.0.1/libexec/vendor/lib/python2.7/site-packages/twisted/internet/defer.py", line 306, in addCallbacks
    self._runCallbacks()
--- <exception caught here> ---
  File "/usr/local/Cellar/ooniprobe/2.0.1/libexec/vendor/lib/python2.7/site-packages/twisted/internet/defer.py", line 588, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/usr/local/Cellar/ooniprobe/2.0.1/libexec/lib/python2.7/site-packages/ooni/ui/web/server.py", line 131, in _notify
    assert not d.called, "Deferred is already called"
exceptions.AssertionError: Deferred is already called
```

--- a/ooni/deck/deck.py
+++ b/ooni/deck/deck.py
@@ -150,6 +150,10 @@ class NGDeck(object):
         if self.metadata.get('no_collector', False):
             self.no_collector = True
 
+        if (self.no_collector is False and
+                self.bouncer.backend_type == "onion"):
+            self.requires_tor = True
+
     @property
     def tasks(self):
         return self._tasks

--- a/ooni/ui/web/server.py
+++ b/ooni/ui/web/server.py
@@ -529,7 +529,6 @@ class WebUIAPI(object):
     @requires_true(attrs=['_is_initialized'])
     @defer.inlineCallbacks
     def api_measurement_summary(self, request, measurement_id):
-        log.warn("SUMMARY")
         try:
             measurement = get_measurement(measurement_id)
         except InsecurePath:

